osdsctl/cli/dock.go

```diff
@@ -43,9 +43,40 @@ var dockListCommand = &cobra.Command{
 	Run:   dockListAction,
 }
 
+var (
+	dockLimit       string
+	dockOffset      string
+	dockSortDir     string
+	dockSortKey     string
+	dockId          string
+	dockCreatedAt   string
+	dockName        string
+	dockUpdatedAt   string
+	dockDescription string
+	dockStatus      string
+	dockStorageType string
+	dockEndpoint    string
+	dockDriverName  string
+)
+
 func init() {
+	dockListCommand.Flags().StringVarP(&dockLimit, "limit", "", "50", "the number of ertries displayed per page")
+	dockListCommand.Flags().StringVarP(&dockOffset, "offset", "", "0", "all requested data offsets")
+	dockListCommand.Flags().StringVarP(&dockSortDir, "sortDir", "", "desc", "the sort direction of all requested data. supports asc or desc(default)")
+	dockListCommand.Flags().StringVarP(&dockSortKey, "sortKey", "", "id", "the sort key of all requested data. supports id(default), name, status, endpoint, drivername, description")
+	dockListCommand.Flags().StringVarP(&dockId, "id", "", "", "list docks by id")
+	dockListCommand.Flags().StringVarP(&dockCreatedAt, "createdAt", "", "", "list docks by created time")
```

Comment:
Does this list docks by the exact created time?  It is more useful to list docks created before or after a timestamp.

Same comments applied to other timestamp filters.

Reply:
Only supports accurate query or not support this parameter. Unlike SQL statements, if you want to achieve this function, you need to achieve a very very complex implementation

Reply:
Please remove the filtering using timestamps such as createdAt and updatedAt.  We can look at filtering by a range later.

Reply:
Got that